speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/ConfigTests.cpp

```diff
@@ -71,14 +71,13 @@ END_TEST_COMBINATORIAL3(IndicatorTest, booleans, 2, booleans, 2, booleans, 2)
 void PControlTest(bool p_steer, bool p_gas, bool p_brake, bool p_intervention, bool p_force)
 {
     SDAConfig config;
-    tParticipantControl arr = {p_steer, p_gas, p_brake, p_intervention, p_force};
+    tParticipantControl arr = {p_steer, p_gas, p_brake, p_intervention};
```

Comment:
now p_force is unused, so remove it (from function, documentation and usages of function)